<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VConfig</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::EditPaste"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>217</width>
      <height>238</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="colorscheme">
       <item>
        <widget class="QLabel" name="colorscheme_label">
         <property name="text">
          <string>Colorscheme:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>Blue</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dark Blue</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Default</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Delek</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Desert</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Elflord</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Evening</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="lineNumbers">
         <property name="text">
          <string>Line numbers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="fileTypeDetection">
         <property name="text">
          <string>File type detection</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="fileTypeIndent">
         <property name="text">
          <string>File type indendation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="syntaxHighlight">
         <property name="text">
          <string>Syntax highlighting</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="noBackupFiles">
         <property name="text">
          <string>No backup files</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="wildmenu">
         <property name="text">
          <string>Wildmenu (suggest file names)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="genConfig">
       <property name="text">
        <string>Generate Vim Config</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
